sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -57,45 +76,65 @@
               >
             </status-info-label>
           </div>
-          <content-group-info>
-            <div>
-              <span class="label-bold">Earliest date of disbursement: </span>
-              <span>{{
-                assessmentAwardData.estimatedAward.disbursement1Date
-              }}</span>
-            </div>
-            <div
-              v-if="
-                isFirstDisbursementCompleted &&
-                assessmentAwardData.estimatedAward.disbursement1DocumentNumber
-              "
-            >
-              <span class="label-bold">Certificate number: </span>
-              <span>{{
-                assessmentAwardData.estimatedAward.disbursement1DocumentNumber
-              }}</span>
-            </div>
-          </content-group-info>
         </content-group>
       </v-col>
       <v-col>
         <content-group>
           <h3 class="category-header-medium primary-color my-3">Final award</h3>
           <!-- Final award table. -->
-          <award-table
-            :awardDetails="assessmentAwardData.finalAward"
-            :offeringIntensity="assessmentAwardData.offeringIntensity"
-            identifier="disbursementReceipt1"
-            v-if="showFirstFinalAward"
-          />
-          <div v-else>
+          <template v-if="showFirstFinalAward">
```

Comment:
Is there a reason for using template here instead of div?

Reply:
Is there a reason to use a div here instead of a template 😄 
Unless a `div` is required the template would allow the conditional display in the same way.
The bottom line would be, that we should not add a `div` only for the sake of hiding/showing something. Does it make sense?

Reply:
Please take a look in the official documentation from Vue from where I got the point.
https://vuejs.org/guide/essentials/conditional.html#v-if-on-template.

_Because v-if is a directive, it has to be attached to a single element. But what if we want to toggle more than one element? In this case we can use v-if on a <template> element, which serves as an invisible wrapper. The final rendered result will not include the <template> element._